File: app/src/main/java/practicaltask/riseapps/com/practicaltask/ui/regionList/RegionListView.java
package practicaltask.riseapps.com.practicaltask.ui.regionList;

import java.util.List;

import practicaltask.riseapps.com.practicaltask.data.enums.Region;
import practicaltask.riseapps.com.practicaltask.ui.base.BaseView;

public interface RegionListView extends BaseView {

    void showRegionList(List<Region> regions);

    void openCountryListOfRegion(Region region);
}

File: app/src/main/java/practicaltask/riseapps/com/practicaltask/ui/regionList/RegionListView.kt
package practicaltask.riseapps.com.practicaltask.ui.regionList

import practicaltask.riseapps.com.practicaltask.data.enums.Region
import practicaltask.riseapps.com.practicaltask.ui.base.BaseView

interface RegionListView : BaseView {

    fun showRegionList(regions: List<Region>)

    fun openCountryListOfRegion(region: Region)
}
